python/elastic_search_serialization.py

```diff
@@ -570,8 +570,9 @@ def scan(
     )
     items = []
     for item in response:
-        item[ID] = item[_SOURCE][INDEX][ID]
-        item[_SOURCE].pop(INDEX, None)
+        if item[_SOURCE].get(INDEX, None):
```

Comment:
What if `_SOURCE` is missing, wouldn't it be better to have the following if expression
```
if _SOURCE in item and INDEX in item[_SOURCE]:
```
🤔 *the same applies in the next changed block

Reply:
@gitbuda good catch - updated with extra check on _SOURCE key